init.go

```diff
@@ -130,26 +130,21 @@ func Init(root, dir string, force bool) error {
 
 	logger.Debug().
 		Str("configFile", stackfile).
-		Msg("Create stack file.")
-	f, err := os.Create(stackfile)
+		Msg("Create new configuration.")
```

Comment:
We can probably reduce duplication of debug statements here by using a sub-logger when `Str("configFile", stackfile)` is first introduced a bit further up in the function. I missed this when adding logging.